showcase/app/templates/components/icon.hbs

```diff
@@ -42,46 +42,71 @@
     </SF.Item>
   </Shw::Flex>
 
-  <Shw::Divider @level={{2}} />
+  <Shw::Divider />
 
   <Shw::Text::H2>Color</Shw::Text::H2>
 
+  <Shw::Text::H4 @tag="h3">Color inheritance</Shw::Text::H4>
```

Comment:
Nit: you could just do `<Shw::Text::H3>Color inheritance</Shw::Text::H4>`

Reply:
This should be changed to H3 as to not skip a heading level

Reply:
Missed that the @tag was changed initially. If you want the visual style of the smaller H4 here for some reason then this is fine.

Reply:
> Nit: you could just do <Shw::Text::H3>Color inheritance</Shw::Text::H4>

`<Shw::Text::H4 @tag="h3">` means render an `<h3>` tag with the visual style of a `.shw-text-h4`